README.md

```diff
@@ -9,31 +9,63 @@ view-git-file macro on any Confluence page.
 ## Marketplace
 
 You can install Git for Confluence by using the Universal Plugin Manager inside
-Confluence, or by visiting the [Git for Confluence marketplace entry][1]
+Confluence, or by visiting the
```

Comment:
Why?

Reply:
I've added the link directly.

Reply:
Dat zie ik, maar waarom? Ik leer altijd graag mee.

Reply:
Omdat een gebruiker nu direct op de link kan klikken ipv naar de footnotes moet scrollen, daar een link moet selecteren en in zijn browser moet plakken.

Reply:
Die link wordt gewoon inline gerenderd, daar verandert niets aan toch?

Reply:
Eerst stonden de links als footnote, nu staan de links inline wat mij beter leek dan ze als footnote toevoegen. De footnotes zijn in de laatste commit ook verwijderd. Het doel hier is het voor een gebruiker zo simpel mogelijk maken om naar de docu te gaan. Als een gebruiker eerst nog naar de footnotes moet zoeken is dat weer een extra stap die ik graag wil voorkomen.